src/tests/system/tests/test_authentication.py

```diff
@@ -0,0 +1,78 @@
+""" SSSD Sanity Test Cases
+
+:requirement: IDM-SSSD-REQ : KRB5 Provider
```

Comment:
I don't think the requirement is correct here as the tests do not only check krb5 auth provider but also plain ldap.

Reply:
Now the requirement should be correct.